packages/app/src/cli/models/extensions/specifications/function.test.ts

```diff
@@ -90,6 +90,8 @@ describe('functionConfiguration', () => {
         },
         enable_creation_ui: true,
         module_id: moduleId,
+        localization: {},
```

Comment:
is it ok to send an empty object or do we want to not send the localization key at all if there aren't localization files?

Reply:
This looks like the default for other extensions to send an empty `{}` once the other changes are merged this should be fine.